packages/docs/src/components/layout/Layout.tsx

```diff
@@ -112,10 +112,10 @@ const Layout = ({
               {children}
             </article>
           ) : (
-            <article className="ds-u-padding-x--3 ds-u-sm-padding-x--6 ds-u-sm-padding-bottom--6 ds-u-padding-top--1 ds-u-padding-bottom--3 ds-u-md-display--flex page-body">
+            <article className="ds-u-padding-x--3 ds-u-sm-padding-x--6 ds-u-sm-padding-bottom--6 ds-u-padding-top--1 ds-u-padding-bottom--3 page-body">
```

Comment:
This `article` has 1 direct child, a `div` with `ds-l-row`. It's unknown what the `ds-u-md-display--flex` class was adding. Removing fixes the overflow issues on the `pre` tag and doesn't appear to have any negative effects.